crates/factor-outbound-networking/src/config.rs

```diff
@@ -74,6 +85,32 @@ pub fn validate_service_chaining_for_components(
     Ok(())
 }
 
+/// If the components left in an app after subsetting no longer require
```

Comment:
what does "subsetting" mean here?

Reply:
"subsetting" = "trimming an app down to a subset of its prior components".  Let me see if I can find a clearer word.

Reply:
reworded